sirepo/template/radia.py

```diff
@@ -584,6 +608,11 @@ def _field_lineout_plot(sim_id, name, f_type, f_path, beam_axis, v_axis, h_axis)
     )
 
 
+def _find_obj_by_id(obj_arr, obj_id):
```

Comment:
For me this is easier to read. May not be the case for all

```python
def _find_obj_by_id(obj_arr, obj_id):
  return next(filter(lambda o: o.id == obj_id, obj_arr), None)
```

Reply:
Hm, I like the list comprehension better. With your indulgence I'd like to table this for discussion in the dev meeting

Reply:
Sure thing. 

I also noticed that _find_obj_by_id() and _find_obj_by_name() consist of the exact same code. Maybe create _find_obj_by_key() and pass in name or id